### ISSUE TITLE
cat targets.txt | aquatone showing no command found

### ISSUE THREAD

**Issue by robocyber** [michenriksen/aquatone#154]
I think I am asking silly que but there is lack of any new tuts around since the new code came.
I am running the binary ./aquatone then it show started at (date)..
after that cat targets.txt | aquatone  will show no commands found.
What silly thing I am missing?

**Comment by mshassy** [comment]
use 
cat targets.txt | ./aqautone

**Comment by robocyber** [comment]
cat targets.txt | sudo ./aqautone starts it up but will only show the subdomain like
EX. in targets.txt =example.com
result- https://example.com, http://example.com

**Comment by mshassy** [comment]
What's the format of your target.txt?

**Comment by robocyber** [comment]
> What's the format of your target.txt?

example.com

**Comment by mshassy** [comment]
> 
> 
> > What's the format of your target.txt?
> 
> example.com

Do you want to enumerate subdomains? The new aquatone doesn't do subdomain enumeration I suppose.

**Comment by robocyber** [comment]
> > > What's the format of your target.txt?
> > 
> > 
> > example.com
> 
> Do you want to enumerate subdomains? The new aquatone doesn't do subdomain enumeration I suppose.

yes, sorry what then it does now! Should we use with combination of other tools.
then what does this means https://github.com/michenriksen/aquatone#giving-aquatone-data

**Comment by mshassy** [comment]
> 
> 
> Sure that now means its only purpose is to screenshot sites now.

Yes. This is extracted from here. https://michenriksen.com/blog/aquatone-now-in-go/

No DNS enumeration

Yes, Aquatone is now completely focused on screenshotting and reporting. I know a lot of people used Aquatone for its DNS enumeration capabilities and it was definitely very good at that when it was released. Now other tools are doing a much better job of this, so I decided to leave it out of the new Aquatone, and instead make it easy to use it with your tool of choice.
